code/modules/surgery/encased.dm

```diff
@@ -6,214 +6,226 @@
 
-		if (!hasorgans(target))
-			return 0
+/datum/surgery_step/open_encased/can_use(mob/living/user, mob/living/carbon/human/target, target_zone, obj/item/tool,datum/surgery/surgery)
 
-		var/obj/item/organ/external/affected = target.get_organ(target_zone)
-		return affected && !(affected.status & ORGAN_ROBOT) && affected.encased && affected.open >= 2
+	if (!hasorgans(target))
+		return 0
+
+	var/obj/item/organ/external/affected = target.get_organ(target_zone)
+	return affected && !(affected.status & ORGAN_ROBOT) && affected.encased && affected.open >= 2
 
 
 /datum/surgery_step/open_encased/saw
+	name = "saw bone"
 	allowed_tools = list(
 	/obj/item/weapon/circular_saw = 100, \
 	/obj/item/weapon/melee/energy/sword/cyborg/saw = 100, \
 	/obj/item/weapon/hatchet = 75
 	)
 
-	min_duration = 50
 	max_duration = 70
 
-	can_use(mob/living/user, mob/living/carbon/human/target, target_zone, obj/item/tool)
-		if (!hasorgans(target))
-			return
-		var/obj/item/organ/external/affected = target.get_organ(target_zone)
-		return ..() && affected && affected.open == 2
+/datum/surgery_step/open_encased/saw/can_use(mob/living/user, mob/living/carbon/human/target, target_zone, obj/item/tool,datum/surgery/surgery)
+	if (!hasorgans(target))
+		return
+	var/obj/item/organ/external/affected = target.get_organ(target_zone)
+	return ..() && affected && affected.open == 2
+
+/datum/surgery_step/open_encased/saw/begin_step(mob/living/user, mob/living/carbon/human/target, target_zone, obj/item/tool,datum/surgery/surgery)
 
-	begin_step(mob/user, mob/living/carbon/human/target, target_zone, obj/item/tool)
+	if (!hasorgans(target))
+		return
+	var/obj/item/organ/external/affected = target.get_organ(target_zone)
 
-		if (!hasorgans(target))
-			return
-		var/obj/item/organ/external/affected = target.get_organ(target_zone)
+	user.visible_message("[user] begins to cut through [target]'s [affected.encased] with \the [tool].", \
+	"You begin to cut through [target]'s [affected.encased] with \the [tool].")
+	target.custom_pain("Something hurts horribly in your [affected.name]!",1)
+	..()
 
-		user.visible_message("[user] begins to cut through [target]'s [affected.encased] with \the [tool].", \
-		"You begin to cut through [target]'s [affected.encased] with \the [tool].")
-		target.custom_pain("Something hurts horribly in your [affected.name]!",1)
-		..()
+/datum/surgery_step/open_encased/saw/end_step(mob/living/user, mob/living/carbon/human/target, target_zone, obj/item/tool,datum/surgery/surgery)
 
-	end_step(mob/living/user, mob/living/carbon/human/target, target_zone, obj/item/tool)
+	if (!hasorgans(target))
+		return
+	var/obj/item/organ/external/affected = target.get_organ(target_zone)
 
-		if (!hasorgans(target))
-			return
-		var/obj/item/organ/external/affected = target.get_organ(target_zone)
+	user.visible_message("\blue [user] has cut [target]'s [affected.encased] open with \the [tool].",		\
+	"\blue You have cut [target]'s [affected.encased] open with \the [tool].")
```

Comment:
Spans. There's more lines that need these instead of color macros but at this point I'll just stop posting more comments, as this PR already loads slowly enough on Github.